src/webapp/public/locales/de/translation.json

```diff
@@ -58,6 +61,12 @@
         },
         "timers": {
           "description": "Wähle die Anzahl der Minuten nachdem die Aktion ausgeführt werden soll."
+        },
+        "synchronisation": {
+            "description": "Wähle den zu setzenden Status.",
+            "label-toggle": "Umschalten",
+            "label-true": "Aktivieren",
+            "label-false": "Deaktivieren"
```

Comment:
Aah.. `True` means `activate` .. you should call it that way in your code as well

Reply:
changed the options to "enable" and "disable" intead "true" and "false" for better readability